### DISCUSSION TITLE
high cpu using

### DISCUSSION THREAD

**Discussion by nguyenhathienphu** [Chia-Network/chia-blockchain#8784] [Q&A]
in ver 1.2.7, cpu use is about 20-30%, but when i update to 1.2.9, cpu is alway 45-60 %,
how to fix. ??

**Comment by rmica** [comment]
My boxes (ubuntu) stay between 90 and 99 % idle. Didn't notice any changes upgrading to 1.2.9.

**Comment by BrandtH22** [comment]
@nguyenhathienphu and @rmica , the newest version(s) of chia has fixes in place to resolve this issue. Please download it from here: https://www.chia.net/downloads/

Since this issue has been resolved with the latest version(s) of chia we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
